api/server/app.cc

```diff
@@ -476,24 +475,27 @@ static int InitCgroup(int multiplier) {
 		    s = std::to_string(std::atoi(s.c_str()) * multiplier);
 		    out << s << std::endl;
 	    }
-	    in.close();
-	    out.close();
     } else {
 	    LOG_WARNING_("can't set cgroup priority");
     }
-    
+
+    // new cgroup use a diferent directory for cpushare and cpu
+    // if not, we need to initilize some files
     fileToRead = std::string(cgroupPath) + "/cpuset.mems";
-    std::string msgLog;
-    if (access(fileToRead.c_str(), R_OK)){
-	    printf("access to file OK  \n");
+    if (!access(fileToRead.c_str(), R_OK)){
```

Comment:
I am confused about this, 
If I understand well the bash code is that if the file dosen't exist we will try to set cgroup cpuset.mems from the readFile that we try to verify if it is exist ? 
anyway if am following this logic the file exist so we don't even run this close ??